### DISCUSSION TITLE
Approving PRs

### DISCUSSION THREAD

**Discussion by callumrollo** [GliderToolsCommunity/GliderTools#106] [Q&A] [answered]
What is the process for doing this in GliderTools? We have a [PR template](https://github.com/GliderToolsCommunity/GliderTools/blob/master/.github/pull_request_template.md), but I'm unsure what the mechanism is for approving PRs. Who can approve a PR? How do we manage PR review assignment?

**Comment by soerenthomsen** [comment]
Hi @callumrollo maybe you could link the PR template in your text?

**Reply by callumrollo** [reply]
Done, my question is more about what happens after the PR is submitted though

**Reply by soerenthomsen** [reply]
Sure got it. Just could not find the pull request (PR) template! Merci!

**Comment by jbusecke** [comment, marked as answer]
Hi @callumrollo,

Currently the settings are configured that a PR requires an approving review at the minimum and then any of the admins (e.g. @lukegre @dhruvbalwada me) can merge it.

Once we have a solid test base we can perhaps expand the admins, but the test really are crucial so that any admin can trust that the package will work.

Ultimately I think any PR that is not a very small change should be reviewed by someone with experience on the code. This slows the process but IMO always leads to better quality code in the end.

Does that answer your question? Also very happy for suggestions. I just modified these settings ad-Hof and we can modify them again.

**Reply by callumrollo** [reply]
Thanks @jbusecke, that clarifies it. Good to keep the review process. I'll focus on getting a good set of tests operational before working on the other issues
